allennlp/service/servable/models/bidaf.py

```diff
@@ -39,3 +32,25 @@ def predict_json(self, inputs: JSONDict) -> JSONDict:
         output_dict["span_end_probs"] = output_dict["span_end_probs"].tolist()
 
         return output_dict
+
+    @classmethod
+    def from_config(cls, config: Params) -> 'BidafServable':
+        dataset_reader_params = config.pop("dataset_reader")
+        assert dataset_reader_params.pop('type') == 'squad'
+        dataset_reader = SquadReader.from_params(dataset_reader_params)
+
+        serialization_prefix = config.pop('serialization_prefix')
+        vocab_dir = os.path.join(serialization_prefix, 'vocabulary')
+        vocab = Vocabulary.from_files(vocab_dir)
+
+        model_params = config.pop("model")
+        assert model_params.pop("type") == "bidaf"
+        model = BidirectionalAttentionFlow.from_params(vocab, model_params)
```

Comment:
Hmm...  This seems like the wrong way to load models.  You shouldn't have to instantiate the vocab separately, and _then_ load the model from params; you should be able to just load a model from a config file, without any additional parameters (where the difference between this and what we have now is that `from_params` is how you need to construct it at _training_ time, whereas if you have an already-trained model, you shouldn't have to do it the same way).

How you've done this is correct for how the code is now; I'm just saying that we should make a better way to do this soon, both for this and for the `evaluate` command.  @DeNeutoy, you agree?